bin/asl-setup-dkms-mok

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/bash
+#
+
+if [ "${USER}" != "root" ]; then
```

Comment:
It might be a little more idiomatic to check the value of UID.
```suggestion
if [ "${UID}" -ne 0" ]; then
```

Reply:
or check ${EUID}